src/lib/merkle_mask/masking_merkle_tree.ml

```diff
@@ -1012,6 +1012,15 @@ module Make (Inputs : Inputs_intf.S) = struct
                 add_location () )
       | Some location ->
           Ok (`Existed, location)
+
+    let all_accounts_on_masks { parent; maps; _ } =
```

Comment:
Sorry, saw this just now - I think this should `assert_is_attached`? I'm realizing the previous PR might have wanted to do that too.

Reply:
I don't find trouble on implementing this for unattached masks, but I can add them if needed